hoomd/mpcd/validation/validate_at_test.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2009-2019 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+# Maintainer: mphoward
+
+import unittest
+import numpy as np
+
+import hoomd
+from hoomd import md
+from hoomd import mpcd
+
+class mpcd_at_validation(unittest.TestCase):
+    def setUp(self):
+        hoomd.context.initialize()
+
+        # box size L and solvent density 5
+        L = 20
+        self.density = 5.
+
+        # solute initially on simple cubic lattice
+        self.solute = hoomd.init.create_lattice(hoomd.lattice.sc(a=1.0), L)
+        snap = self.solute.take_snapshot(all=True)
+        if hoomd.comm.get_rank() == 0:
+            snap.particles.mass[:] = self.density
+        self.solute.restore_snapshot(snap)
+
+        # at
+        self.solvent = mpcd.init.make_random(N=int(self.density*L**3), kT=1.0, seed=42)
+        mpcd.integrator(dt=0.1)
+        self.at = mpcd.collide.at(seed=791, period=1, kT=1.0)
+        mpcd.stream.bulk(period=1)
+        md.integrate.nve(hoomd.group.all())
+
+    def test_solvent(self):
+        """Test momentum conservation for AT solvent."""
+
+        # initial momentum of both should be zero
+        slv = self.solvent.take_snapshot()
+        slt = self.solute.take_snapshot()
+        if hoomd.comm.get_rank() == 0:
+            slv_p0 = np.sum(slv.particles.velocity, axis=0)
+            slt_p0 = self.density*np.sum(slt.particles.velocity, axis=0)
+            np.testing.assert_array_almost_equal(slv_p0, [0,0,0])
```

Comment:
The numpy documentation recommends `assert_allclose` over `assert_array_almost_equal`. 

To compare to 0, use `assert_allclose(a, b, atol=1e-6)` throughout.